### PR TITLE
Issue 5 pagination watering log

### PR DESCRIPTION
**Pull request by Tomaszabc**
Pagy added and some styling
obraz

### CODE REVIEW

File: app/controllers/plants_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def index
 
   # GET /plants/1 or /plants/1.json
   def show
+    @pagy, @watering_logs = pagy(@plant.watering_logs.order(created_at: :desc), items: 3)
```
Comment (Ptrboro) [inline app/controllers/plants_controller.rb]: Do we need `items: 3`? 🤔 I think it does nothing. If you want to adjust the number of records per page the correct option is `limit: 3`. 

💡 you can also do something like `limit: params[:limit] || 3` - then FE can decide on how many records to display on a single page. It is a common pattern, e.g. in e-commerce stores. But we don't use it in our case so there is no need for that 👍

File: config/initializers/pagy.rb [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+# Pagy initializer file (9.0.5)
+# Customize only what you really need and notice that the core Pagy works also without any of the following lines.
+# Should you just cherry pick part of this file, please maintain the require-order of the extras
+
+# Pagy Variables
+# See https://ddnexus.github.io/pagy/docs/api/pagy#variables
+# You can set any pagy variable as a Pagy::DEFAULT. They can also be overridden per instance by just passing them to
+# Pagy.new|Pagy::Countless.new|Pagy::Calendar::*.new or any of the #pagy* controller methods
+# Here are the few that make more sense as DEFAULTs:
+Pagy::DEFAULT[:limit] = 5                    # default
```
Comment (Ptrboro) [inline config/initializers/pagy.rb]: I think setting a global default of 5 records might not be a good idea 🤔 It's very little and it's better to keep the default more sensible and override them only when needed. Let's set it to what was before (20?) and explicitly set `limit: 5` when we call `pagy` in our controller.

File: lib/tasks/beautify_html.rake [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  desc "Beautify all HTML files in the project"
+  task :beautify do
+    Dir.glob("app/views/**/*.html.erb").each do |file|
+      original = File.read(file)
+      beautified = HtmlBeautifier.beautify(original)
+      File.open(file, "w") { |f| f.write(beautified) }
+      puts "Beautified: #{file}"
+    end
+  end
+end
```
Comment (Ptrboro) [inline lib/tasks/beautify_html.rake:13]: Nice stuff, I like it! 👏 

But a tip for the future, it is better to do such changes in separate PRs so it is easier to review :)

Comment (Tomaszabc) [reply lib/tasks/beautify_html.rake:13]: Ok. Chat GPT did this :)

File: app/controllers/plants_controller.rb (hunk 1 above)
Comment (Tomaszabc) [reply app/controllers/plants_controller.rb]: Yes, this is not needed. It was for testing purposes and I forgot to clear this.
Typically this behavior is handled by pagy.rb initializer.

obraz

File: config/initializers/pagy.rb (hunk 2 above)
Comment (Tomaszabc) [reply config/initializers/pagy.rb]: If you think this is better, it will be like that:

obraz

**Review by Ptrboro — APPROVED** [review]
LGTM! 👍

**Pull request merged**